next-tavla/src/Shared/utils/converters.ts

```diff
@@ -20,7 +20,18 @@ export function convertSettingsVersion(settings: any): TSettings {
     }
 }
 
-const versions = [V2, V1] as const
+const versions = [V3, V2, V1] as const
+
+export function V3(setting: ReturnType<typeof V2>) {
+    const newSetting = {
+        ...setting,
+        theme: setting.theme === 'default' ? ('entur' as const) : setting.theme,
+    }
+
+    if (!newSetting.theme) delete setting.theme
```

Comment:
Hva er hensikten med å `delete` setting.theme? ser ikke ut som man bruker setting.theme til noe etter delete

Reply:
Jeg mistenker at hensikten her er å sørge for at man ikke sitter igjen med et undefined felt i newSetting, og at `delete` egentlig skulle slette newSetting.theme? Dette vil nok fungere, men jeg hadde foretrukket at man droppet å sette newSetting sitt theme til undefined i utgangspunktet. Kunne man fått til dette på en måte?